src/Microsoft.AspNet.Razor.Runtime/TagHelpers/TagHelperRunner.cs

```diff
@@ -26,7 +38,9 @@ public class TagHelperRunner
                 executionContext.Items,
                 executionContext.UniqueId,
                 executionContext.GetChildContentAsync);
-            var tagHelperOutput = new TagHelperOutput(executionContext.TagName, executionContext.HTMLAttributes)
+            var tagHelperOutput = new TagHelperOutput(executionContext.TagName,
+                                                      executionContext.HTMLAttributes,
```

Comment:
nit: don't add the extra spacing, just let it lie 1 indent in (that's where it goes when you just hit enter on the previous line.